Add-ons/Umbraco-Deploy/Upgrades/version-specific.md

```diff
@@ -38,6 +49,22 @@ For example, to use LocalDb, you would place this in your `appSettings.json` con
 }
 ```
 
+Or set the configuration value of `Umbraco:Deploy:Settings:PreferLocalDbConnectionString` to `true`:
+
+```json
+{
+    "Umbraco": {
+        "Deploy": {
+            "Settings": {
+                "PreferLocalDbConnectionString": true
+            }
+        }
+    }
+}
+```
+
+If you are upgrading from Umbraco 9 and already have a LocalDb instance, setting this value to `true` will ensure it is used rather than a new, empty Sqlite database.
```

Comment:
```suggestion
If you are upgrading from Umbraco 9 and already have a LocalDB instance, setting this value to `true` will ensure it is used rather than a new, empty SQLite database.
```